x/pki/x509/x509_test.go

```diff
@@ -65,3 +86,75 @@ func Test_VerifyRootCertificate(t *testing.T) {
 	err := certificate.Verify(certificate)
 	require.Nil(t, err)
 }
+
+func Test_FormatOID(t *testing.T) {
```

Comment:
Consider splitting this method for VID and PID

Reply:
Done! ✅